app/controllers/articles_controller.rb

```diff
@@ -32,6 +32,7 @@ def new
   def create
     @article = Article.new(article_params)
     if @article.save
+      @article.subscribe_author(current_user)
```

Comment:
With the recent addition of the author select box on the article form, yes. We need to account for the possibility of `current_user` not ending up as the `article.author` on `articles#create` and `articles#update`:

![image](https://cloud.githubusercontent.com/assets/65950/10197581/693955f8-6766-11e5-9561-243b5c7b5b60.png)